apptainer/moose-dev.def

```diff
@@ -65,6 +71,21 @@ From: {{ APPTAINER_FROM }}
     WASP_GIT_REMOTE={{ WASP_GIT_REMOTE }}
     export MOOSE_JOBS={{ MOOSE_JOBS }}
 
+    # Eventually, this will go in the base container as far down as
+    # we can. But we don't want to do a full rebuild at the moment,
+    # so this is good for now.
+    mv ${ROOT_BUILD_DIR}/set_prompt.bash  ${SINGULARITY_ROOTFS}{{ SET_PROMPT_SCRIPT }}
+    cat <<'EOF' > ${SINGULARITY_ROOTFS}/.singularity.d/env/99-zzz_prompt.sh
+if [ -n "${CUSTOM_PROMPT}" ]; then
+    PS1=${CUSTOM_PROMPT}
+else
+    if [ -e {{ SET_PROMPT_SCRIPT }} ]; then
+        source {{ SET_PROMPT_SCRIPT }}
+        set_prompt
```

Comment:
I _think_ this needs to #
```suggestion
        PS1="\$(set_prompt)"
```

Reply:
actually not needed, since set_prompt sets PS1=